integrations/extensions/starter-kits/coveo/README.md

```diff
@@ -39,73 +41,74 @@ If you want to add this starter kit to an _existing_ assistant, you cannot use t
 
-![Search failed](./assets/search_failed.png)<br> 
+    - Click "New Step" and change "without conditions" to "with conditions" and set "Ran successfully" to "false", and set "And then" to "End the action".  Then add the following to the "Assistant says": "Sorry. The search failed. Please try again another time."
 
+    ![Search failed](./assets/search-failed.png)<br>
 
-- Add a "New Step", then:
-    - Change "without conditions" to "with conditions" and select "true" for "Ran successfully"
-    - Click the fX button to add variables and add all the following new session variables, replacing `step_123_result_1` with the actual variable name and selecting "Expression" each time. For context on why we do this and what these mean, see [Extensions Made Easy with Watson Assistant Starter Kits](https://medium.com/ibm-watson/extensions-made-easy-with-watson-assistant-starter-kits-6b177f624697):
-```
-link0 = ${step_123_result_1}.body.items.get(0).clickUri
-title0 = ${step_123_result_1}.body.items.get(0).title
-snippet0 = ${step_123_result_1}.body.items.get(0).excerpt
-link1 = ${step_123_result_1}.body.items.get(1).clickUri
-title1 = ${step_123_result_1}.body.items.get(1).title
-snippet1 = ${step_123_result_1}.body.items.get(1).excerpt
-link2 = ${step_123_result_1}.body.items.get(2).clickUri
-title2 = ${step_123_result_1}.body.items.get(2).title
-snippet2 = ${step_123_result_1}.body.items.get(2).excerpt
-```
+    - Still in the "Search" action, add another "New Step".  In the new step:
+    - In "Assistant says" hit `$` and select "Ran Successfully" and then click on `</>` in the upper right of that box to see the full JSON for the response.  In there, you should see a field called `variable` with a value that looks something like `step_123_result_1`.  Copy that value.
+    - Click "abc" in the upper right and delete the variable in "Assistant says" (we only put it there to copy the variable name).
+    - Change "without conditions" to "with conditions" and select "true" for "Ran successfully" and "${step_123_result_1.body.results}.size > 0"
+    - Click on "Variable values" and set `search_results` to "${step_123_result_1.body.results}". This allows you to pass the search results to action that displays them.
+    - Set "And then" to "Go to another action" and select "Show search results" and "Continue" upon return.
 
-![Map variables to results](./assets/search_passed.png)<br>
+      ![Pass results to display action](./assets/pass-results-to-display.png)<br>
 
-- Click "New Step" and also change "without conditions" to "with conditions" and select "Ran successfully" is "true" and "link0" is "defined".  Then add the following to the "Assistant says":
+    - Click "New Step", change "without conditions" to "with conditions", and set "Ran successfully" to "true" and "{step_123_result_1.results}.size > 0". Also set "And then" to "End the action". This step clears the search results to help prevent exceeding result size limitations.
```

Comment:
This step is missing the part where you set the `search_results` variable to the expression `${step_123_result_1}.results.body = []`.  That's shown in the picture, but not mentioned in the text.

Also on this line `{step_123_result_1.results}.size > 0` should be ${step_123_result_1.results.body}.size > 0 -- the `$` and the "body" term are missing.